components/Auction/index.js

```diff
@@ -59,9 +41,26 @@ const HandleAuctions = () => {
       }
     } else {
       setUserMoney(0);
+      return null;
     }
   };
 
+  useEffect(() => {
+    (async () => {
+      await fetchAndSetAuctions();
+      await getUserWallet();
+      await fetchSelfDetails()
+        .then((res) => {
```

Comment:
Please don't mix the `.then().catch()` syntax with async-await if not needed.
You can wrap the block in a try-catch as any of the above requests could also fail 😓